### ISSUE TITLE
docker_container is --env-file support? 

### ISSUE THREAD

**Issue by brandoncormany** [chef-cookbooks/docker#732]
TL:DR does the docker cookbook support `--env-file value`? I see it listed as supported for version 0.34 of this cookbook but do not see any listing of it being removed in the CHANGELOG. 

### Cookbook version
2.5.8

### Chef-client version
12.12.15

### Platform Details
Ubuntu 14.04

### Scenario:
I want to pass in a list of ENV variables into an docker container using pre-compiled JSON file. 

### Steps to Reproduce:
Attempt to start container with an ENV variables loaded from JSON file. Simliar to `docker run --env-file ` command.

### Expected Result:
Container starts with ENV loaded inside container

### Actual Result:
```
==> legacy_lb: ================================================================================
==> legacy_lb: Recipe Compile Error in /var/chef/cache/cookbooks/legacy_nginx/recipes/default.rb
==> legacy_lb: ================================================================================
==> legacy_lb:
==> legacy_lb:
==> legacy_lb: SyntaxError
==> legacy_lb: -----------
==> legacy_lb: /var/chef/cache/cookbooks/legacy_nginx/recipes/default.rb:41: syntax error, unexpected tSTRING_BEG, expecting keyword_do or '{' or '('
==> legacy_lb:
==> legacy_lb:   env-file "/opt/legacy/dynamic-loadbalancer/env.json"
==> legacy_lb:
==> legacy_lb:             ^
==> legacy_lb:
==> legacy_lb:
==> legacy_lb: Platform:
==> legacy_lb: ---------
==> legacy_lb: x86_64-linux
==> legacy_lb:
==> legacy_lb:
==> legacy_lb:
==> legacy_lb: Running handlers:
==> legacy_lb: [2016-07-16T21:18:26+00:00] ERROR: Running exception handlers
==> legacy_lb: Running handlers complete
==> legacy_lb:
==> legacy_lb: [2016-07-16T21:18:26+00:00] ERROR: Exception handlers complete
==> legacy_lb: Chef Client failed. 0 resources updated in 11 seconds
==> legacy_lb: [2016-07-16T21:18:26+00:00] FATAL: Stacktrace dumped to /var/chef/cache/chef-stacktrace.out
==> legacy_lb: [2016-07-16T21:18:26+00:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
==> legacy_lb: [2016-07-16T21:18:26+00:00] ERROR: /var/chef/cache/cookbooks/legacy_nginx/recipes/default.rb:41: syntax error, unexpected tSTRING_BEG, expecting keyword_do or '{' or '('
==> legacy_lb:   env-file "/opt/legacy/dynamic-loadbalancer/env.json"
==> legacy_lb:             ^
==> legacy_lb: [2016-07-16T21:18:26+00:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
```

**Comment by gmreburn** [comment]
env-file support was lost when the project structure was reorganized. You can you env that until someone makes a PR. It is an unordered array.

```
docker_container 'env' do
  repo 'debian'
  env ['PATH=/usr/bin', 'FOO=bar']
  command 'env'
  action :run_if_missing
end
```

**Comment by brandoncormany** [comment]
I figured something like that happened. thanks @gmreburn 

I did work around this by creating an array and then using this array with env attribute.
